### PR TITLE
added 104 + old PR from last week that was never merged

### PR DESCRIPTION
**Pull request by Davidutro**

### CODE REVIEW

File: governance/governance-and-risk-meetings/summaries/episode-103.md [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+
+<https://youtu.be/eIL6rpL54-s>
+
+## Introduction
+
+### LongForWisdom
+
+[00:00](https://youtu.be/eIL6rpL54-s)
+
+- Hello, everyone, welcome to 103rd MakerDAO Scientific Governance and Risk meeting. Today is Thursday, July 30th. My name is LongForWisdom. I am one of the MakerDao Governance Facilitators. I am joined by a host of people who are interested in Maker things.
```
Comment (amy-jung) [inline governance/governance-and-risk-meetings/summaries/episode-103.md:27]: MakerDAO*

Comment (Davidutro) [reply governance/governance-and-risk-meetings/summaries/episode-103.md:27]: good catch!

**Comment by Davidutro** [comment]
@amy-jung fixed

File: governance/communications/governance-communications-mandate-draft.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+---
+tags: Maker Relay, Communications team
+---
 # MakerDAO Communications Team Mandate Draft
```
[the remark this answers is not in the document]
Comment (Davidutro) [reply governance/communications/governance-communications-mandate-draft.md:4]: Ok removing it from this PR

File: governance/communications/maker-relay-ep-00.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+---
+tags: Maker Relay
+---
 ![](./assets/maker-relay-ep-0.png)
 
 Hey Maker community, welcome to the first edition of Maker Relay!
```
[the remark this answers is not in the document]
Comment (Davidutro) [reply governance/communications/maker-relay-ep-00.md:6]: Next commit will have it.

File: governance/communications/maker-relay-ep-01.md [hunk 4]
```diff
@@ -1,3 +1,6 @@
+---
+tags: Maker Relay
+---
 # Maker Relay 01
```
[the remark this answers is not in the document]
Comment (Davidutro) [reply governance/communications/maker-relay-ep-01.md:4]: fixed in next commit

**Comment by Davidutro** [comment]
@amy-jung Fixed!
Ready for merge hopefully

**Pull request merged**